### PR TITLE
Init repo

### PR DESCRIPTION
**Pull request by bjankord**
Initial repo setup. This PR is based on [this PR](https://github.com/cerner/terra-enzyme-intl/pull/1) minus the files that are not needed for initial terra repo setup. 

This will allow us to focus on [this PR](https://github.com/cerner/terra-enzyme-intl/pull/1) without having to sift through the 9000+ additions for all the boilerplate code.

Edit: The travis CI build is going to fail on this PR as there are no Jest tests included. This will be resolved in #1

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Cerner Corporation
+
+- Matt Henkes [@mjhenkes]
+- Cody Price [@dev-cprice]
```
Comment (emilyrohrbough) [inline CONTRIBUTORS.md:4]: Add yourself as collaborator

File: package.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    "extends browserslist-config-terra"
+  ],
+  "eslintConfig": {
+    "extends": "terra"
+  },
+  "files": [
+    "lib"
+  ],
+  "dependencies": {
+    "browserslist-config-terra": "^1.1.0"
```
Comment (emilyrohrbough) [inline package.json]: This should be a dev dependency.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    "react-intl": ">= 2.7"
+  },
+  "scripts": {
+    "compile": "babel src --out-dir lib --copy-files",
+    "danger": "danger ci",
+    "jest": "jest --config jestConfig.js",
+    "lint": "eslint --ext .js,.jsx .",
+    "precompile": "rm -rf lib",
+    "prepublishOnly": "npm run compile",
+    "pretest": "npm run lint",
```
Comment (emilyrohrbough) [inline package.json]: Lets remove pretest script and add it to the test script

Comment (bjankord) [reply package.json]: https://github.com/cerner/terra-enzyme-intl/pull/2/commits/013c120bcea4351d2810bcaaa14d5acde14dc233

File: package.json (hunk 2 above)
Comment (bjankord) [reply package.json]: https://github.com/cerner/terra-enzyme-intl/pull/2/commits/013c120bcea4351d2810bcaaa14d5acde14dc233

File: CONTRIBUTORS.md (hunk 1 above)
Comment (bjankord) [reply CONTRIBUTORS.md:4]: https://github.com/cerner/terra-enzyme-intl/pull/2/commits/013c120bcea4351d2810bcaaa14d5acde14dc233

**Pull request merged**